src/components/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -92,7 +92,7 @@ denoted by an `aria-current` attribute value of "page".
 - [W3 Breadcrumbs Practice](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html)
 - [Chakra UI Breadcrumb](https://chakra-ui.com/docs/components/navigation/breadcrumb)
 
-## Long Titles
+## Long Titles shorten to 60 characters, an ellipsis will be added, and that the Tooltip is used to see the entire text.
```

Comment:
"Long Titles" is the title. Your description should go below it and add more context to it. For example, "Breadcrumb text titles should be short and concise. When they are over 60 characters...." etc.